.github/workflows/zephyr.yml

```
@@ -21,8 +21,8 @@ jobs:
           imx8 imx8x imx8m,
           # - IPC4 default
           mtl,
-          # Very few IPC3 platforms support IPC4 too.
-          -i IPC4 tgl,
+          # Some IPC3 platforms support IPC4 too.
+          -i IPC4 tgl, -i IPC4 tgl-h,
```

For a shorter list:
```suggestion
          -i IPC4 tgl tgl-h,
```
I will submit that as a followup patch, it's more urgent to merge this as is.